<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>A3PreviewDataDialog</class>
 <widget class="QDialog" name="A3PreviewDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A3P Export Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wFilter" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filter Kolom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbKolom">
        <item>
         <property name="text">
          <string>Semua</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Klien</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bahan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jumlah Page</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Jumlah Repeat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Keterangan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Total Klik</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leKolomFilter"/>
      </item>
      <item>
       <widget class="QToolButton" name="tbKolomFilter">
        <property name="text">
         <string>Terapkan</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Filter Tanggal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbTanggal">
        <item>
         <property name="text">
          <string>Semua</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Baris Maksimum per halaman</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbRow">
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maksimum</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="mainTable">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wNav" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbFirst">
        <property name="text">
         <string>First</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbPrev">
        <property name="text">
         <string>Prev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lPaging">
        <property name="text">
         <string>Page 1 / 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbNext">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbLast">
        <property name="text">
         <string>Last</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="tbSave">
        <property name="text">
         <string>Simpan Perubahan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbClose">
        <property name="text">
         <string>Tutup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbPrint">
        <property name="text">
         <string>Export Excel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leKolomFilter</sender>
   <signal>returnPressed()</signal>
   <receiver>tbKolomFilter</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
